roles/nuage_master/vars/main.yaml

```diff
@@ -15,3 +15,6 @@ nuage_ca_master_rest_server_key: "{{ nuage_mon_rest_server_crt_dir }}/nuageMonSe
 nuage_ca_master_rest_server_crt: "{{ nuage_mon_rest_server_crt_dir }}/nuageMonServer.crt" 
 
 nuage_master_crt_dir : /usr/share/nuage-openshift-monitor
+cspadminpasswd: ''
+adminsusername: 'admin'
+adminuserpasswd: 'admin'
```

Comment:
Could we add `nuage_master_` to the front of these variable names so we can quickly know which role makes use of them?